packages/components/src/highlight-cards/weekly-highlight-cards.tsx

```diff
@@ -60,16 +149,7 @@ export default function WeeklyHighlightCards( {
 	const translate = useTranslate();
 
 	const textRef = useRef( null );
-	const settingsActionRef = useRef( null );
-	const [ isTooltipVisible, setTooltipVisible ] = useState( false );
-	const [ isPopoverVisible, setPopoverVisible ] = useState( false );
-
-	// @TODO: Set the popover to disappear when the user clicks outside of the popover.
-	const togglePopoverMenu = useCallback( () => {
-		setPopoverVisible( ( isVisible ) => {
-			return ! isVisible;
-		} );
-	}, [] );
+	const [ isTooltipVisible, setTooltipVisible ] = useState( true );
```

Comment:
With this change, opening Jetpack Stats on the live branch automatically opens the tooltip next to the 7-day highlights header. It only goes away after being moused over once.

<img width="1254" alt="image" src="https://github.com/Automattic/wp-calypso/assets/4044428/03d65730-fe64-4efa-b794-04c83ca6f5dd">

```suggestion
	const [ isTooltipVisible, setTooltipVisible ] = useState( false );
```

Reply:
Good catch! I should have removed it after debugging.